eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Binary/CostModelsSpec.hs

```diff
@@ -9,30 +10,43 @@
 module Test.Cardano.Ledger.Alonzo.Binary.CostModelsSpec (spec) where
 
 import Cardano.Ledger.Alonzo.Core
-import Cardano.Ledger.Alonzo.Scripts
 import Cardano.Ledger.BaseTypes
 import Cardano.Ledger.Binary (ToCBOR (..))
 import Cardano.Ledger.Binary.Decoding
 import Cardano.Ledger.Binary.Encoding
+import Cardano.Ledger.Plutus.CostModels
 import Cardano.Ledger.Plutus.Language
 import qualified Data.Map as Map
+import Data.Proxy
 import Data.Text (Text)
-import Data.Typeable
 import Data.Word (Word8)
 import Lens.Micro
 import Test.Cardano.Ledger.Alonzo.Arbitrary
 import Test.Cardano.Ledger.Common
 
-spec ::
-  forall era.
-  AlonzoEraPParams era =>
-  Proxy era ->
-  Spec
-spec pxy = do
-  describe "CostModels CBOR deserialization" $ do
-    validCostModelProp @era pxy
+spec :: forall era. AlonzoEraPParams era => Spec
+spec = do
+  describe "CBOR deserialization" $ do
+    validCostModelProp @era Proxy
     invalidCostModelProp @era Proxy
     unknownCostModelProp @era Proxy
+  prop "applyPPUpdates" $ \valid validUpdate unknown unknownUpdate -> do
```

Comment:
The formatting looks a bit strange here, don't know why fourmolu doesn't pick it up, but it looks like it should be aligned with the line above `unKnownCostModelProp @era Proxy`, no?

Reply:
No, this was intentional, since this property has nothing to do with serialization. It is just part of the `CostModels` `Spec`

Reply:
Oh yes, makes sense!  Missed the content of "describe"